.npmrc

```
@@ -1,4 +1,4 @@
-registry=http://registry.npmjs.com/
+registry=http://registry.npmjs.org/
```

Can we also make this https?

```suggestion
registry=https://registry.npmjs.org/
```